### ISSUE TITLE
Keyboard navigation in popup goes through hidden elements (report broken site/share forms)

### ISSUE THREAD

**Issue by ghostwords** [EFForg/privacybadger#3105]
https://github.com/EFForg/privacybadger/pull/2629#issuecomment-646101693

**Comment by DelfinSR** [comment]
Hi!

Is this solved/being solved on a pr?? Just checking to start working on it :))

**Comment by DelfinSR** [comment]
Hi! I've been doing some research on how to do this and before starting I wanted to tell my plan to a more experienced contributor. I think the best solution is to extract the report broken site/share forms to a separate html file and change the view when the buttons are pressed (not with z-index and opacity). Did you thought of a better way of doing this??

**Comment by ghostwords** [comment]
I don't think we should separate the forms and introduce navigating to other documents into the mix because that kind of means rebuilding the forms from scratch (more work, could introduce new issues), and I'm not sure we need to give up on the current approach.

Instead, we should be able to fix keyboard navigation by changing precisely how the forms get hidden and shown. Rather than updating opacity, we could instead change the forms' display property. However, we will then lose the subtle transition we have now when you switch between the regular popup and the nested form. Could we fix keyboard navigation while preserving some sort of transition effect?
